src/components/Checkbox/Checkbox.tsx

```diff
@@ -28,41 +26,45 @@ export const Checkbox = ({
   defaultChecked = false,
   helperText,
   inputRef,
-  isDisabled = false,
+  disabled = false,
```

Comment:
Is this prop necessary or can it be deleted a la [buttons](https://github.com/cfpb/design-stories/pull/112)?

Reply:
It needs to stay because we are passing that attribute separately to the `<input>` element, while the the rest of the props are passed to the wrapper `<div>`